cypress/page/personal.page.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable require-jsdoc */
+interface PersonalInform {
+  name: string;
+  lastName: string;
+  email: string;
+  gender: string;
+  mobileNumber: string;
+  hobbies: string[];
+  subject: string;
+  currentAddress: string;
+  state: string;
+  city: string;
+}
+
+class PersonalPage {
+  private toolsQaFormURL: string;
+  private firstName: string;
+  private lastName: string;
+  private email: string;
+  private gender: string;
+  private mobileNumber: string;
+  private hobbies: string;
+  private subject: string;
+  private currentAddresss: string;
+  private submit: string;
+  private submitting: string;
+  private state: string;
+  private city: string;
+  private modalFirstName: string;
+
+  constructor() {
+    this.toolsQaFormURL = "https://demoqa.com/automation-practice-form";
+    this.firstName = "#firstName";
+    this.lastName = "#lastName";
+    this.email = "#userEmail";
+    this.gender = "#genterWrapper input";
+    this.mobileNumber = "#userNumber";
+    this.hobbies = "#hobbiesWrapper .col-md-9";
+    this.subject = ".subjects-auto-complete__value-container";
+    this.currentAddresss = "#currentAddress";
+    this.state = "#state";
+    this.city = "#city";
+    this.submit = "#userForm";
+    this.submitting = ".modal-title";
+    this.modalFirstName = "tr:nth-child(10)  td:nth-child(2)";
+  }
+
+  public completTheProcess(message: string): any {
```

Comment:
instead of completTheProcess, it should be better to say `verifyConfirmationPageTitle(message: string)` or `verifyConfirmationModalTitle(message: string)`